src/screens/Home/Home.tsx

```diff
@@ -1,30 +1,83 @@
 import * as React from 'react';
 import { LatestPostsQuery } from '../../generated/graphql';
 import { Link } from 'react-router-dom';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import DiscussionCard from '../../components/DiscussionCard'
+import styled from 'styled-components';
+
 
 interface Props {
-  data: LatestPostsQuery;
+  data: LatestPostsQuery
+  className?: string
 }
 
-const className = 'Home';
-
-const Home: React.FC<Props> = ({ data }) => (
-  <div className={className}>
-    <h3>Latest posts</h3>
-    <ul className={`${className}__list`}>
-      {!!data.posts &&
-        data.posts.map(
-          (post) =>
-            !!post && (
-              <li key={post.id} className={`${className}__item`}>
-                <Link to={`/post/${post.id}`}>
-                  {post.author.username} ({post.creation_date}): {post.title}
-                </Link>
-              </li>
-            ),
-        )}
-    </ul>
-  </div>
+// const className = 'Home';
+const Container = styled.div`
+  ul {
+    padding: 0;
+  }
+
+  li {
+    list-style-type: none;
+  }
+
+  h3 {
+    font-family: 'Roboto Mono';
+    font-weight: 500;
+    font-size: 30px;
+    color: #222;
+    margin-bottom: 20px;
+  }
+
+  .Home__item {
+    margin: 0 0 10px 0;
+    border: 1px solid #EEE;
+  }
+
+  .Home__item:hover {
+    border: 1px solid #BBB;
+    text-decoration: none;
+  }
+
+  .Home__item a:hover {
+    text-decoration: none;
+  }
+`;
+
+const Home = ({ data, className }: Props) => (
+  <Container className='Home'>
+      <h3>Latest posts</h3>
+      <Row>
+        <Col md={6} lg={8}>
+          <ul className='Home__list'>
+            {!!data.posts &&
+              data.posts.map(
+                (post) =>
+                  !!post && (
+                    <li key={post.id} className='Home__item'>
+                      {<Link to={`/post/${post.id}`}>
+                        <DiscussionCard 
+                          title={post.title}
+                          author={post.author.username}
+                          creation_date={post.creation_date}
+                          replies={post.replies.length.toString()}
+                        >
+                        </DiscussionCard>
```

Comment:
Since we have no children here, no need for an opening and closing tag.
We can call the discussion card:
```js
<DiscussionCard 
  title={post.title}
  author={post.author.username}
  creation_date={post.creation_date}
  replies={post.replies.length.toString()}
/>
```